lib/logger.js

```diff
@@ -1,15 +1,14 @@
 const winston = require('winston')
 const options = require('./options')
-const _ = require('lodash')
+const _ = require('lodash/fp')
 
 const logger = new winston.Logger({
   level: options.logLevel,
   transports: [
     new (winston.transports.Console)({ timestamp: true, colorize: true })
   ],
   rewriters: [
-    (...[,, meta]) => _.hasIn('toString', meta)
-      ? meta.toString() : 'Error, no further information is available'
```

Comment:
'Error, no further information is available' isn't actually needed, I'm not sure why it made sense on the last review, maybe because my other mistakes were silly ones.

Reply:
I think we were concerned that we'd get huge error strings again, if winston tried to print an object. Can you confirm that won't happen?

Reply:
The main cause for huge js dump were errors from axios. I'm sanitizing only instances of `Error` which are the most common objects passed to the logger, any other object will get fully printed. 

The issue with the meta check is that not every message will have a meta. For instance:
```
logger.error('this is a error')
logger.error('this is a error', null)

// meta === {} or null depending on the second parameter
// msg === 'this is a error'
(...[level, msg, meta]) =>...
```

We would be appending `[object Object]` or `Error, no further information is available` to a lot of valid message strings.

Reply:
Ah right.